blocks/carousel/carousel.css

```diff
@@ -0,0 +1,173 @@
+.carousel .carousel-slides-container {
+  position: relative;
+}
+
+.carousel .carousel-slides,
+.carousel .carousel-slide-indicators {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.carousel .carousel-slides {
+  display: flex;
+  scroll-behavior: smooth;
+  scroll-snap-type: x mandatory;
+  overflow-x: scroll;
+  overflow-y: clip;
+}
+
+.carousel .carousel-slides::-webkit-scrollbar {
+  display: none;
+}
+
+.carousel .carousel-slide {
+  flex: 0 0 100%;
+  scroll-snap-align: start;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: center;
+  position: relative;
+  width: 100%;
+  min-height: 40rem;
```

Comment:
I already regret this, but could we make this more dynamic? `max(40rem, 100vh - var(--nav-height))`, so it's never taller than the viewport (unless it _has_ to be because of text)?

Reply:
fwiw, that 40rem was very arbitrary.

I tested what you have above, and it didn't seem to work, what about `min-height: minmax(40rem, calc(100vh - var(--nav-height)));`

Reply:
n/m `minmax` doesn't work either. Just `min` I think is what we want which is what I have now